enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.unit.spec.tsx

```diff
@@ -0,0 +1,37 @@
+import { renderWithProviders } from "__support__/ui";
+import { SdkContentWrapper } from "embedding-sdk/components/private/SdkContentWrapper";
+import {
+  createMockSettingsState,
+  createMockState,
+} from "metabase-types/store/mocks";
+
+describe("SdkContentWrapper", () => {
+  it("injects the font-face declaration when available", () => {
+    const state = createMockState({
+      settings: createMockSettingsState({
+        "application-font-files": [
+          {
+            src: "https://example.com/foo.woff2",
+            fontFormat: "woff2",
+            fontWeight: 700,
+          },
+        ],
+      }),
+    });
+
+    renderWithProviders(<SdkContentWrapper />, { storeInitialState: state });
+
+    const rules = Array.from(document.styleSheets).flatMap(sheet =>
```

Comment:
I never knew you could directly use `document` in the test.

Reply:
God bless react-testing-library and JSDOM!